test/e2e/certificates_test.go

```diff
@@ -354,6 +355,50 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("OCP-65035-CLOUD_CREDENTIALS_SECRET_NAME works well in mint/passthrough clusters", func() {
+		It("should obtain a valid LetsEncrypt certificate", func() {
+
+			By("getting Infrastructure object")
+			infra, err := configClient.Infrastructures().Get(ctx, "cluster", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			if infra.Status.PlatformStatus.Type != configv1.GCPPlatformType {
+				Skip("Skip this case for current plaform, it's not gcp.")
+			}
+
+			By("Check cloud credential in cluster")
+			_, err = loader.KubeClient.CoreV1().Secrets("cert-manager").Get(ctx, "gcp-credentials", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			By("creating CredentialsRequest object")
+			loader.CreateFromFile(testassets.ReadFile, filepath.Join("testdata", "credentials", "credentialsrequest_gcp.yaml"), "")
+
+			By("Configure cert-manager to use credential, setting this credential secret name in subscription object")
+			credentialValue := "gcp-credentials"
+			err = patchSubscriptionWithCloudCredential(ctx, loader, credentialValue)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("creating new certificate ClusterIssuer")
+			clusterIssuerName := "acme-dns01-clouddns-ambient"
```

Comment:
The name of the ClusterIssuer is determined by the `clusterissuer_gcp.yaml` file. Therefore the variable `clusterIssuerName` should be retrieved from the file instead of being assigned the value.

Same issue for the variable `certName := "cert-with-acme-dns01-clouddns-ambient"`.

Reply:
Is there any difference/shortage between retrieved from file and assigning value based on the file which assigned it already? in my view, retrieved from file will add code cost.

Reply:
As talked offline, keep as is, unless Dev shares ideas.